app/views/browse/index.html.erb

```diff
@@ -27,7 +27,11 @@
         <i class="fa fa-arrow-right" aria-hidden="true"></i><span class="sr-only">Next</span>
       <% end %>
       <div class='pull-right'>
-        <%= render 'catalog/view_type_group' %>
+        <%= render(Blacklight::Response::ViewTypeComponent.new(
```

Comment:
We seem to have our own `view_type_group` partial. Should we use it here, instead of that upstream one?

https://github.com/sul-dlss/SearchWorks/blob/main/app/views/catalog/_view_type_group.html.erb#L1-L5

Reply:
I'm up for deleting that local partial if you agree. It's just calling the component, which I'm doing here.

Reply:
Sure, done.